apriltags2_ros/include/apriltags2_ros/common_functions.h

```diff
@@ -64,7 +64,7 @@
 
 #include "apriltags2_ros/AprilTagDetection.h"
 #include "apriltags2_ros/AprilTagDetectionArray.h"
-#include "apriltag.h"
+#include "apriltags2/apriltag.h"
```

Comment:
Where does this change come from? Is it required or optional?

Reply:
I was not able to get the package to build without this change.

Reply:
I tried to build the packages again today and had to remove this for them to build. I don't know what I was doing wrong before. This has been put back to the way it was.